core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-reports/store/dot-experiments-reports-store.ts

```diff
@@ -350,9 +350,8 @@ export class DotExperimentsReportsStore extends ComponentStore<DotExperimentsRep
 
     private parseDaysLabels(labels: Array<string>): string[] {
         return [getPreviousDay(labels[0]), ...labels].map((item) => {
-            const date = new Date(item);
-            const day = date.getDate();
-            const monthTranslated = this.dotMessageService.get(MonthsOfTheYear[date.getMonth()]);
+            const [, month, day] = item.split('-').map(Number);
```

Comment:
put an underscore in the first index, is more visible that is not gonna be used

Reply:
with the underscore lint still cry.